### DISCUSSION TITLE
How does it represent temperature?

### DISCUSSION THREAD

**Discussion by Whale-xh** [jeshraghian/snntorch#261] [Q&A] [answered]
Thank you very much for your contribution in the SNN field. 

When I am researching SNNTORCH to predict temperature, how should I represent the predicted temperature? 

Thank you very much!

**Comment by ahenkes1** [comment, marked as answer]
I assume you want to predict temperature? It is a real number, so you should check out the regression tutorials on the website! Does this help?

**Reply by Whale-xh** [reply]
Thank you for your reply. I have already resolved this issue！
